swaylock/main.c

```diff
@@ -212,7 +212,7 @@ static cairo_surface_t *select_image(struct swaylock_state *state,
 
 static void load_image(char *arg, struct swaylock_state *state) {
 	// [<output>:]<path>
-	struct swaylock_image *image = malloc(sizeof(struct swaylock_image));
+	struct swaylock_image *image = calloc(sizeof(struct swaylock_image), 1);
```

Comment:
This doesn't _actually_ affect anything, but the arguments are in the wrong order.